controllers/servingruntime_controller.go

```diff
@@ -166,26 +192,42 @@ func (r *ServingRuntimeReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	}
 
 	// Reconcile this serving runtime
+	log.Info("==== ChinDebug sr reconcile 2 ====", "req.NamespacedName", req.NamespacedName)
 	rt := &kserveapi.ServingRuntime{}
-	if err = r.Client.Get(ctx, req.NamespacedName, rt); errors.IsNotFound(err) {
-		log.Info("Runtime is not found")
-
-		// remove runtime from info map
-		r.runtimeInfoMapMutex.Lock()
-		defer r.runtimeInfoMapMutex.Unlock()
-
-		if r.runtimeInfoMap != nil {
-			// this is safe even if the entry doesn't exist
-			delete(r.runtimeInfoMap, req.NamespacedName)
+	crt := &kserveapi.ClusterServingRuntime{}
+	var spec *kserveapi.ServingRuntimeSpec
+	var annotations map[string]string
```

Comment:
Per other comment I think we can remove annotations here, and instead of `crt` var at this level can have an `mf.Owner` var which is set to one or the other below (where annotations are currently set).